exonum/src/events/noise/tests.rs

```diff
@@ -28,7 +28,10 @@ use crypto::{
     gen_keypair, gen_keypair_from_seed, x25519::into_x25519_keypair, Seed, PUBLIC_KEY_LENGTH,
 };
 use events::error::into_other;
-use events::noise::{write, Handshake, HandshakeParams, HandshakeResult, NoiseHandshake};
+use events::noise::{
```

Comment:
It isn't related to your changes, but I have notices here and in the adjusted files that some imports are not grouped. Could you please fix it? :upside_down_face: